### DISCUSSION TITLE
Import sandboxie box's from old windows installation

### DISCUSSION THREAD

**Discussion by Aholicknight** [sandboxie-plus/Sandboxie#4157] [Q&A Feedback] [answered]
![SandMan_20240800_cdqJLr2TSE](https://github.com/user-attachments/assets/3a828b36-bfa8-4f6f-926a-f1116b9bbaa2)

Hello. I recently re-installed Windows and was wondering if it's possible to import/restore sandboxie box's. If this is possible, how can I do this? Thank you!

**Comment by love-code-yeyixiao** [comment, marked as answer]
You need to create a new box from zero,then modify the FileRootPath and KeyRootPath to the pervous path.Or if you saved Sandboxie.ini as back files,you can copy the sandbox section into the new Sandboxie.ini,but if not,you have to use a new sandbox configuration.










---Original---
From: ***@***.***&gt;
Date: Tue, Aug 20, 2024 14:51 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [sandboxie-plus/Sandboxie] Import sandboxie box's from old windowsinstallation (Discussion #4157)




 
SandMan_20240800_cdqJLr2TSE.png (view on web)
 
Hello. I recently re-installed Windows and was wondering if it's possible to import/restore sandboxie box's. If this is possible, how can I do this? Thank you!

**Reply by Aholicknight** [reply]
Where would sandboxie.ini be located?

**Comment by love-code-yeyixiao** [comment]
In your "Windows" folder of system volume.



---Original---
From: ***@***.***&gt;
Date: Tue, Aug 20, 2024 15:16 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [sandboxie-plus/Sandboxie] Import sandboxie box's from oldwindows installation (Discussion #4157)




 
Where would sandboxie.ini be located?

**Comment by Aholicknight** [comment]
Replacing the old Sandboxie.ini with the new Sandboxie.ini worked. Thank you!
